app/views/sections/_cookie-acceptance.html.erb

```diff
@@ -18,7 +18,11 @@
 
 
             <div class="dialog__buttons">
-              <a href="#" class="button call-to-action-button" data-action="click->cookie-acceptance#accept" data-cookie-acceptance-target="agree" id="cookies-agree">
+              <!--
+                The id is `biscuits-agrees` instead of `biscuits-agree`
+                to prevent ad blockers from removing the button.
```

Comment:
Do you mean ` The id is biscuits-agree instead of cookies-agree`?

Reply:
Whoops, yep I fixed that but didn't push it 🤦